lesson-17/index.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+
+// HW-17
+
+// task 1
+
+// const a = ("'true'");
+// const b = Boolean(NaN);
```

Comment:
Тут нужно присвоить значение false. Зачем такой необычный способ? Может я что-то упустил?
```suggestion
// const b = false;
```